include/wx/generic/private/listctrl.h

```diff
@@ -695,7 +695,7 @@ class wxListMainWindow : public wxWindow
     }
 
     wxImageList* GetSmallImageList() const
-        { return m_small_image_list; }
+        { return m_small_images ? m_small_images->GetImageList() : nullptr; }
```

Comment:
I think we should get rid of this function, it seems wrong that it's still used.